### PR TITLE
Dvora

### PR DESCRIPTION
**Pull request by DvoraSchwartz**

### CODE REVIEW

**Review by jamberusimantov — COMMENTED** [review]
ctrl c ctrl v

**Review by orgolestani — COMMENTED** [review]
Finish form component

File: client/src/components/Form.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Form = () => {
+    return (
+        <React.Fragment>
```
Comment (yonatanpa) [inline client/src/components/Form.ts:5]: Refactor <React.Fragment> => <></>

File: client/src/components/Layout/Header.tsx [hunk 2]
```diff
@@ -7,4 +7,6 @@ const Header = () => {
         </header>
     )
 }
-export default Header;
\ No newline at end of file
+export default Header;
+
```
Comment (yonatanpa) [inline client/src/components/Layout/Header.tsx:11]: Remove empty line

**Review by yonatanpa — CHANGES_REQUESTED** [review]

**Comment by DvoraSchwartz** [comment]
Updated

**Pull request closed without merging**